internal/jobs/twitter/auth.go

```diff
@@ -27,47 +21,13 @@ func NewScraper(config AuthConfig) *Scraper {
 
 	scraper := &Scraper{Scraper: newTwitterScraper()}
 
-	// Configure whether to skip login verification
-	scraper.SetSkipLoginVerification(config.SkipLoginVerification)
-
 	// Try loading cookies
 	if err := LoadCookies(scraper.Scraper, config.Account, config.BaseDir); err == nil {
 		logrus.Debugf("Cookies loaded for user %s.", config.Account.Username)
-		if scraper.IsLoggedIn() {
-			logrus.Debugf("Already logged in as %s.", config.Account.Username)
-			return scraper
-		}
+		scraper.SetBearerToken()
+		return scraper
 	} else {
-		logrus.Warnf("Failed to load cookies for user %s: %v", config.Account.Username, err)
-	}
-
-	RandomSleep()
-
-	if err := scraper.Login(config.Account.Username, config.Account.Password, config.Account.TwoFACode); err != nil {
-		logrus.WithError(err).Warnf("Login failed for %s", config.Account.Username)
+		logrus.WithError(err).Warnf("Failed to load cookies for user %s", config.Account.Username)
 		return nil
 	}
-
-	RandomSleep()
-
-	if err := SaveCookies(scraper.Scraper, config.Account, config.BaseDir); err != nil {
-		logrus.WithError(err).Errorf("Failed to save cookies for %s", config.Account.Username)
-	}
-
-	logrus.Debugf("Login successful for %s", config.Account.Username)
-	return scraper
-}
-
-func (scraper *Scraper) Login(username, password string, twoFACode ...string) error {
-
-	var err error
-	if len(twoFACode) > 0 {
-		err = scraper.Scraper.Login(username, password, twoFACode[0])
-	} else {
-		err = scraper.Scraper.Login(username, password)
-	}
-	if err != nil {
-		return fmt.Errorf("login failed: %v", err)
-	}
-	return nil
```

Comment:
I love the smell of deleted code in the morning. Just assuming that this isn't actually needed! ;-)

Reply:
It basically doesn't work anymore - login via creds is only exposed to enterprise-level apps now, which are an insane amount of money.  I don't mind that miners have to manage their own cookies if they go this route.